test/Microsoft.Framework.Configuration.FileExtensions.Test/FileConfigurationBuilderExtensionsTest.cs

```diff
@@ -54,8 +54,17 @@ public void GetBasePath_ReturnEmptyIfNotSet()
             // Act
             var actualPath = configurationBuilder.GetBasePath();
 
+            string expectedPath = string.Empty;
+
+#if DNXCORE50
+             expectedPath = AppContext.BaseDirectory;
+#else
+            expectedPath = AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY") as string ??
+                AppDomain.CurrentDomain.BaseDirectory ?? string.Empty;
+#endif
+
```

Comment:
Funny tests, but I guess this protects us from changing the implementation without changing the test :smile: